Jenkinsfile

```diff
@@ -138,29 +138,34 @@ if (env.CHANGE_TITLE && env.CHANGE_TITLE.contains('[ci-skip]')) {
-                                        //            variable: 'VISDIFF_GH_TOKEN',
-                                        //    ]]) {
-                                        //    withEnv([
-                                        //        "VISDIFF_PR_ID=${env.CHANGE_ID}",
-                                        //    ]) {
-                                        //        dir("visdiff") {
-                                        //            sh "npm install"
-                                        //        }
-                                        //        sh "npm install ./visdiff"
-                                        //        dir("desktop") {
-                                        //            sh "../node_modules/.bin/keybase-visdiff 'merge-base(origin/master, HEAD)...HEAD'"
-                                        //        }
-                                        //    }}}
-                                        //}
+                                        if (env.CHANGE_ID) {
+                                            wrap([$class: 'Xvfb', screen: '1280x1024x16']) {
+                                            withCredentials([[$class: 'UsernamePasswordMultiBinding',
+                                                    credentialsId: 'visdiff-aws-creds',
+                                                    usernameVariable: 'VISDIFF_AWS_ACCESS_KEY_ID',
+                                                    passwordVariable: 'VISDIFF_AWS_SECRET_ACCESS_KEY',
+                                                ],[$class: 'StringBinding',
+                                                    credentialsId: 'visdiff-github-token',
+                                                    variable: 'VISDIFF_GH_TOKEN',
+                                            ]]) {
+                                            withEnv([
+                                                "VISDIFF_PR_ID=${env.CHANGE_ID}",
+                                            ]) {
+                                                dir("visdiff") {
+                                                    sh "npm install"
+                                                }
+                                                sh "npm install ./visdiff"
+                                                try {
+                                                    timeout(time: 10, unit: 'MINUTES') {
+                                                        dir("desktop") {
+                                                            sh "../node_modules/.bin/keybase-visdiff 'merge-base(origin/master, HEAD)...HEAD'"
+                                                        }
+                                                    }
+                                                } catch (e) {
+                                                    helpers.slackMessage("#ci-notify", "warning", "<@mgood>: visdiff failed: <${env.BUILD_URL}|${env.JOB_NAME} ${env.BUILD_DISPLAY_NAME}>")
```

Comment:
maybe we should post to its own channel so we can join to come in / out vs having to edit the code?